From 454e639ac7b7279020545116ed55aef31f67b7bf Mon Sep 17 00:00:00 2001
From: Thomas Hautier <thauti.dev@protonmail.com>
Date: Fri, 23 Nov 2018 00:50:34 +0100
Subject: [PATCH] Uboot: Add a copy of u-boot.sd.bin to the root folder

---
 Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile b/Makefile
index 5947cbb7df..8f2a077b53 100644
--- a/Makefile
+++ b/Makefile
@@ -981,6 +981,7 @@ ifeq ($(CONFIG_AML_CRYPTO_IMG), y)
 	@cp -f $(FIP_FOLDER_SOC)/boot.img.encrypt $(FIP_FOLDER)/boot.img.encrypt
 endif
 	@cp -f $(FIP_FOLDER_SOC)/u-boot.* $(FIP_FOLDER)/
+	@cp -f $(FIP_FOLDER_SOC)/u-boot.bin.sd.bin $(srctree)
 	@rm -f $(FIP_FOLDER_SOC)/bl2_new.bin $(FIP_FOLDER_SOC)/boot_new.bin
 	@echo '$(FIP_FOLDER_SOC)/u-boot.bin build done!'
 
@@ -1500,6 +1501,7 @@ distclean: mrproper
 	@rm -f $(FIP_FOLDER_SOC)/u-boot.bin.* $(FIP_FOLDER_SOC)/*.encrypt
 	@rm -f $(FIP_FOLDER)/u-boot.bin.* $(FIP_FOLDER)/*.bin $(FIP_FOLDER)/*.encrypt
 	@rm -f $(srctree)/fip/aml_encrypt_gxb
+	@rm -f $(srctree)/u-boot.bin.sd.bin
 
 backup:
 	F=`basename $(srctree)` ; cd .. ; \
-- 
2.19.1

